recipes/gst-plugins-base/all/conanfile.py

```diff
@@ -85,7 +85,6 @@ def configure(self):
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
-        self.options['gstreamer'].shared = self.options.shared
```

Comment:
Not sure to understand the rational of this modification. You want to raise by default when gst-plugin-base is shared? It means that shared gst-plugin-base packages won't be generated in conan-center.

Reply:
yeah, it's not recommended practice, but it means no shared in CCI, that's very unfurtunate...